[ecsutil] Update ECSUtil v1.0.7.9

bobk2824 opened the pull request:
Update to ECSUtil v1.0.7.9

Triplets supported:
x86-windows
x86-windows-static
x86-windows-static-md
x64-windows
x64-windows-static
x64-windows-static-md

Commit:
Update ECSUtil v1.0.7.9

NancyLi1013 commented on ports/ecsutil/portfile.cmake:
```diff
@@ -28,8 +28,8 @@ endif()
 vcpkg_from_github(
```
`include(vcpkg_common_functions)` is no longer needed.
Could you please remove it?

NancyLi1013 replied:
```
if(VCPKG_TARGET_ARCHITECTURE STREQUAL "x86")
    set(PLATFORM x86)
elseif(VCPKG_TARGET_ARCHITECTURE STREQUAL "x64")
    set(PLATFORM x64)
else()
    message(FATAL_ERROR "Unsupported architecture")
endif()
if(VCPKG_CMAKE_SYSTEM_NAME)
    message(FATAL_ERROR "Unsupported platform. ECSUTIL currently only supports windows desktop.")
endif()
```
These codes might be updated as `vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "UWP" "Linux" "OSX")`.

bobk2824 replied:
Thanks. I've made the above changes. I had to leave this:

> if(VCPKG_TARGET_ARCHITECTURE STREQUAL "x86")
>     set(PLATFORM x86)
> endif()

If that isn't in it will set the platform to "WIN32", which is incorrect.

Commit:
Complete PR requested changes
remove: include(vcpkg_common_functions)
use: vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "UWP" "Linux" "OSX")

bobk2824 commented:
Thanks. I've committed the requested changes.

Commit:
added: Supports: windows&(x64|x86)

bobk2824 commented:
Done. Thanks, Nancy.

NancyLi1013 commented:
Hi @bobk2824 
Thanks for this PR.
It looks good to me now.

bobk2824 commented:
Thanks @NancyLi1013

strega-nil commented:
Looks good, thanks @bobk2824 !!!
